app/Platform/Actions/BuildGameShowPagePropsAction.php

```diff
@@ -76,7 +76,8 @@ public function execute(Game $game, ?User $user): GameShowPagePropsData
             ->filter(
                 fn ($game) => !str_contains($game->title, '[Subset')
             )
-            ->sortBy('sort_title');
+            ->sortBy('sort_title')
+            ->sortByDesc(fn ($game) => $game->achievements_published > 0);
```

Comment:
I've verified the behavior, but this reads as if the sort title is still the primary sort. Aren't sorts processed in order?